portfolio/src/main/webapp/index.html

```diff
@@ -2,21 +2,21 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a cryptic quote from the Secret History:</p>
-      <button onclick="addRandomGreeting()">Hmm...</button>
-      <p></p>
-      <img src="/images/3d.jpeg"
-        alt = "3d-glasses"
-        height = "500">
-      <div id="greeting-container"></div>
+      <h1 class="header">Portfolio</h1>
+      <p class="description">This is Carolyn Wang.</p>
+      <div id ="photo">
+          <img src="/images/3d.jpeg"
```

Comment:
The indentation is 4 spaces instead of 2 spaces.

Reply:
Done. Also fixed indentation in other parts (the Cloud Shell Editor doesn't seem to have a way to automatically fix formatting issues).